lua/astrocommunity/editing-support/nvim-neoclip/init.lua

```diff
@@ -0,0 +1,14 @@
+return {
+  "AckslD/nvim-neoclip.lua",
+  event = { "User AstroFile" },
```

Comment:
```suggestion
  event = { "User AstroFile", "InsertEnter" },
```

This will allow the plugin to also work on an initial new file which would require text to be inserted to have something to yank

Reply:
good idea, thanks